README.md

```diff
@@ -1 +1,85 @@
-# yoti-face-capture-ios
+# YotiFaceCapture
+
+YotiFaceCapture uses the front facing camera to perform face detection, then validates frames where a face has been detected according to Yoti's DeepFas specifications and finally, it returns a result that will either contain a cropped image containing a face or an error.
```

Comment:
I think we need someone business side to phrase this better.

Reply:
yep and not sure we want to mention DeepFas, it is an internal name